src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  min-height: 100vh;
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  padding: 0 50px;
+}
+
+.main-nav {
+  display: flex;
+}
+
+.main-nav-list {
+  display: flex;
+  align-self: stretch;
+  list-style: none;
+  /* margin: 0 15px; */
+}
+
+.logo {
+  display: flex;
+  align-self: center;
+  margin: 10px 0;
+}
+
+.list-item {
+  display: flex;
+  align-self: center;
+  padding: 0 10px;
+}
+
+.list-item:last-child {
+  padding-right: 0;
+}
+
+.links {
+  display: flex;
+  align-self: center;
+  line-height: 14.06px;
+  position: relative;
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.links:hover {
+  color: #00acdc;
+}
+
+.links.is-active {
+  color: #00acdc;
+}
```

Comment:
you could combine these selectors by placing the line under two selectors using comma